dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -204,6 +202,25 @@ const textSyndicationButton = (format: ArticleFormat): string => {
 };
 
 const textArticleLink = (format: ArticleFormat): string => {
+	if (format.design === ArticleDesign.DeadBlog) {
```

Comment:
I like how you've separated out the requirements for deadblogs, I can see how we could refactor (to reduce lines of code) but I prefer your approach :)